tests/Install-Scripts.Test/GivenThatIWantToInstallDotnetFromAScript.cs

```diff
@@ -64,7 +64,8 @@ public class GivenThatIWantToInstallDotnetFromAScript : IDisposable
                 ("release/2.1.8xx", "2\\.1\\.8.*", Quality.None),
                 ("release/2.2.4xx", "2\\.2\\.4.*", Quality.None),
                 ("release/3.0.1xx", "3\\.0\\.1.*", Quality.None),
-               // ("release/3.1.4xx", "3\\.1\\.4.*", Quality.None),
+                // version is outdated
+                // ("release/3.1.4xx", "3\\.1\\.4.*", Quality.None),
```

Comment:
Can you elaborate on what exactly is outdated? 3.1.4xx is an LTS. 3.0.1xx isn't an LTS but it can be downloaded. Sounds like there is a different problem with 3.1.4 builds.

Reply:
please find details here
https://github.com/dotnet/arcade/issues/10026